TransbankAutoservicioSDKConsoleExample/Program.cs

```diff
@@ -0,0 +1,348 @@
+                            print = Console.ReadKey().KeyChar == '1';
+                            Console.WriteLine();
+                            Task<SaleResponse> response = POSAutoservicio.Instance.Sale(amount, ticket, voucher, print);
+                            response.Wait();
+                            Console.WriteLine(response.Result);
+                            break;
+                        }
+                        catch (Exception e)
+                        {
+                            Console.WriteLine("Error: " + e.Message);
+                            break;
+                        }
+
+                    case 6:
+                        try
+                        {
+                            Console.WriteLine("Enter amount");
+                            int.TryParse(Console.ReadLine(), out amount);
+                            Console.WriteLine("Enter ticket number");
+                            ticket = Console.ReadLine();
+                            long commerceCode = SelectCommerceCode();
+                            Console.WriteLine("Select 1 to send voucher in response or any to continue without voucher data:\n");
+                            voucher = Console.ReadKey().KeyChar == '1';
+                            Console.WriteLine();
+                            Console.WriteLine("Select 1 to send intermediate messages or any to continue without intermediate messages:\n");
+                            print = Console.ReadKey().KeyChar == '1';
+                            Console.WriteLine();
+                            Task<MultiCodeSaleResponse> response = POSAutoservicio.Instance.MultiCodeSale(amount, ticket, commerceCode, voucher, print);
+                            response.Wait();
+                            Console.WriteLine(response.Result);
+                            break;
+                        }
+                        catch (Exception e)
+                        {
+                            Console.WriteLine("Error: " + e.Message);
+                            break;
+                        }
+
+                    case 7:
+                        try
+                        {
+                            Console.WriteLine("Select 1 to send voucher in response or any to continue without voucher data:\n");
+                            voucher = Console.ReadKey().KeyChar == '1';
+                            Console.WriteLine();
+                            Task<LastSaleResponse> response = POSAutoservicio.Instance.LastSale(voucher);
+                            response.Wait();
+                            Console.WriteLine(response.Result);
+                            break;
+                        }
+                        catch (Exception e)
+                        {
+                            Console.WriteLine("Error: " + e.Message);
```

Comment:
Maybe use blocks for repetitive code like this one I mean the try catch reutilization. Something like what we use in the Webpay SDK https://github.com/TransbankDevelopers/transbank-sdk-dotnet/blob/master/Transbank/Common/ExceptionHandler.cs

Reply:
👌